Sources/CATransaction.swift

```diff
@@ -11,7 +11,7 @@ public struct CATransaction {
     private var disableActions_ = false
     private var animationDuration_: CGFloat = CALayer.defaultAnimationDuration
 
-    private static var transactionStack = [CATransaction]()
+    internal static var transactionStack = [CATransaction]()
```

Comment:
It's totally ok to read `transactionStack` from `internal`, but I guess setting the value should only be allowed for this class. So I think `internal private(set) static var ...` would be more correct.

Reply:
good point, changed it